### ISSUE TITLE
[Installer] Add dashd

### ISSUE THREAD

**Issue by coolaj86** [webinstall/webi-installers#411]
# Add dashd

We want to add `dashd` because Dash is the best digital currency and `dashd` makes it possible to receive payment notifications and check transaction histories without relying on a 3rd party service.

Ostensibly, it also works across Windows, Mac, and Linux.

This could be as simple as copying `_example`, updating the github releases
info, and updating some config.

Budding cheat sheet at https://github.com/dashhive/dashd-installer/issues/4

# How to create a webi installer

[![Video Tutorial: How to create a webi Installer](https://user-images.githubusercontent.com/122831/91064908-17f28100-e5ed-11ea-9cf0-ab3363cdf4f8.jpeg)](https://youtu.be/RDLyJtiyQHA)

## Skills required

- Basic Command Line knowledge (`mkdir`, `mv`, `ls`, `tar`, `unzip`, variables)

## Steps

1. Clone and setup the webi packages repo
   ```bash
   git clone git@github.com:webinstall/packages.git
   pushd packages/
   npm install
   ```
2. Copy the example template and update with info from Official Releases:
   <https://github.com/___CHANGE/ME___/releases>
   ```bash
   rsync -av _example/ CHANGE-ME/
   ```
   - [ ] update `CHANGE-ME/release.js` to use the official repo
   - [ ] Learn how `CHANGE-ME` unpacks (i.e. as a single file? as a .tar.gz? as
         a .tar.gz with a folder named CHANGE-ME?)
   - [ ] find and replace to change the name
     - [ ] update `CHANGE-ME/install.sh` (see `bat` and `jq` as examples)
     - [ ] update `CHANGE-ME/install.ps1` (see `bat` and `jq` as examples)
3. Needs an updated tagline and cheat sheet
   - [ ] update `CHANGE-ME/README.md`
     - [ ] official URL
     - [ ] tagline
     - [ ] Switch versions
     - [ ] description / summary
     - [ ] General pointers on usage (and perhaps "gotchas")

It's also okay to have multiple people work on part of this (i.e. the Cheat
Sheet can be done independently from the `install.sh`)

**Comment by Akashutreja** [comment]
Hey @coolaj86,
I am a beginner in the open-source space and would love to contribute to getting a head start. please confirm if I can start. 
Thanks

**Comment by curtiscali** [comment]
@coolaj86 I'm looking to contribute as well. I have a bunch of experience programming in industry, but haven't contributed much to OSS. Is this ticket still up for grabs and if so I'd like to get started! Thanks in advance!

**Comment by coolaj86** [comment]
@curtiscali Take a look at https://github.com/webinstall/webi-installer-requests/issues and https://youtu.be/RDLyJtiyQHA and pick something you'd like to work on.

**Comment by coolaj86** [comment]
Closing this specific issue because it already landed, I believe in #619
